### PR TITLE
roachtest: increase `failover` interval to 1 minute

### PR DESCRIPTION
**Pull request by erikgrinaker**
This patch increases the interval between each fail/recover operation from 30 seconds to 1 minute. This gives us better visibility into slower recoveries, such as the upcoming disk stall failure mode.

Epic: none
Release note: None

### CODE REVIEW

File: pkg/cmd/roachtest/tests/failover.go [hunk 1]
```diff
@@ -150,14 +150,14 @@ func runFailoverNonSystem(
 	// the ranges across all nodes regardless.
 	relocateRanges(t, ctx, conn, `database_name = 'kv'`, []int{1, 2, 3}, []int{4, 5, 6})
 
-	// Start workload on n7, using n1-n3 as gateways. Run it for 10 minutes, since
-	// we take ~1 minute to fail and recover each node, and we do 3 cycles of each
+	// Start workload on n7, using n1-n3 as gateways. Run it for 20 minutes, since
+	// we take ~2 minuts to fail and recover each node, and we do 3 cycles of each
```
Comment (tbg) [inline pkg/cmd/roachtest/tests/failover.go:154]: Minutes

**Review by tbg — APPROVED** [review]

**Comment by erikgrinaker** [comment]
bors retry

**Pull request merged**